### PR TITLE
fix(twitter): X support for Twitter

### PR DESCRIPTION
**Pull request by ClaytonTDM**
## 🔧 What does this fix? 🔧

Adds support for x.com

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [contributing guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md).
- [x] I have updated the version appropriately in the `==UserStyle==` header of the `catppuccin.user.css` file.

### CODE REVIEW

File: styles/twitter/catppuccin.user.css
Comment (isabelroses) [inline styles/twitter/catppuccin.user.css:1]: Could this entire change not just be. All I can see here is copy and paste of the entire "twitter.com" code into a new "x.com" domain/
```diff
- @-moz-document domain("twitter.com")
+ @-moz-document domain("twitter.com"), domain("x.com")
```

**Comment by isabelroses** [comment]
superseded by https://github.com/catppuccin/userstyles/pull/895

File: styles/twitter/catppuccin.user.css [hunk 1]
```diff
@@ -1028,6 +1780,143 @@
   }
 }
 
+@-moz-document domain("x.com") {
```
[the remark this answers is not in the document]
Comment (ClaytonTDM) [reply styles/twitter/catppuccin.user.css:1783]: because there was a second twitter.com

File: styles/twitter/catppuccin.user.css
Comment (ClaytonTDM) [reply styles/twitter/catppuccin.user.css:1]: sorry i dont know a lot abt css, i just found not having x support annoying and had to make a pr ;-;

File: styles/twitter/catppuccin.user.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (nonetrix) [reply styles/twitter/catppuccin.user.css:1783]: Your right I missed that, there is one for light version too. I am sorry, but why is it this way anyway? Couldn't it be used twice? I made the PR really late and just assumed it was once in the code, I feel really dumb now that this has been merged

[the remark this answers is not in the document]
Comment (nonetrix) [reply styles/twitter/catppuccin.user.css:1783]: Seems other themes don't do it thing way, seems redundant.

[the remark this answers is not in the document]
Comment (nonetrix) [reply styles/twitter/catppuccin.user.css:1783]: Fixed I am so sorry everyone #898

**Pull request closed without merging**